packages/content/src/lib/markdown.component.ts

```diff
@@ -0,0 +1,63 @@
+import { AsyncPipe } from '@angular/common';
+import {
+  AfterViewChecked,
+  Component,
+  inject,
+  Input,
+  OnInit,
+  OnChanges,
+  ViewEncapsulation,
+} from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { ActivatedRoute, Data } from '@angular/router';
+import { catchError, map, mergeMap, Observable, of, switchMap } from 'rxjs';
+import { ContentRenderer } from './content-renderer';
+
+@Component({
+  selector: 'analog-markdown',
```

Comment:
```suggestion
  selector: 'analogjs-markdown',
```

This is based on te linting rules